co2eq/index.test.js

```diff
@@ -0,0 +1,33 @@
+import * as electricityWorldAverageCarbonModel from './electricity/worldaverage';
+import * as mealCarbonModel from './food/meal';
+import * as ingredientsCarbonModel from './food/ingredients';
+import * as transportationCarbonModel from './transportation/index';
+import * as flightCarbonModel from './flights/index';
+import * as carCarbonModel from './car/index';
+import * as purchaseCarbonModel from './purchase/index';
+import * as energyCarbonModel from './energy/index';
+import * as hotelCarbonModel from './hotelpercountry/index';
+
+const carbonModels = [
```

Comment:
Maybe you want to do this as part of another PR but it would be great if we could create that array somewhere central instead of defining it in:
- tests
- north
- lighthouse

Reply:
And another great point 😄 
See latest commit: https://github.com/tmrowco/northapp-contrib/pull/384/commits/c9adaa6390d30194a2825f244bf6a528ae668435